ichnaea/data/tests/test_export.py

```diff
@@ -36,23 +36,27 @@ def add_reports(
         lat=None,
         lon=None,
         source=None,
+        set_position=True,
     ):
         reports = []
         timestamp = int(time.time() * 1000)
         for i in range(num):
             pos = CellShardFactory.build()
             report = {
                 "timestamp": timestamp,
-                "position": {},
+                "position": None,
                 "bluetoothBeacons": [],
                 "cellTowers": [],
                 "wifiAccessPoints": [],
             }
-            report["position"]["latitude"] = lat or pos.lat
-            report["position"]["longitude"] = lon or pos.lon
-            report["position"]["accuracy"] = 17.0 + i
-            if source is not None:
-                report["position"]["source"] = source
+            if set_position:
```

Comment:
This is the code that creates reports for the export tests. The new parameter ``set_position`` defaults to ``True``, and provides the existing functionality. If set to ``False``, the report has ``"position": None`` and triggers the bug.